cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,26 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    fn process_version(
+        &self,
+        data_dir: &PathBuf,
+    ) -> Fallible<()> {
+        let supported_version = ["1.0.0"];
+        let path = data_dir.join("version");
+        let version = tokio::fs::read(&path)
+            .await
+            .context(format!("Reading {:?}", &path))?;
+        let trimmed_version = version.trim();
+
+        for supported_version in supported_versions.iter() {
+            if supported_version == trimmed_version {
```

Comment:
```suggestion
            if *supported_version == trimmed_version.trim() {
```
`.iter()` returns pointers. Compare value with trimmed version, as in Rust `.trim()` doesn't mutate the string

Reply:
Fixed in c9ef3002 -> 2baa1f2f.